src/js/stores/MarathonStore.js

```diff
@@ -174,11 +177,15 @@ class MarathonStore extends GetSetBaseStore {
           this.emit(MARATHON_GROUP_CREATE_SUCCESS);
           break;
         case REQUEST_MARATHON_GROUP_DELETE_ERROR:
-          let groupErrorMessage = action.data;
-          if (!Object.keys(groupErrorMessage).length) {
-            groupErrorMessage = 'Error destroying group';
+          data = action.data;
+          if (!data || (Util.isObject(data) && !data.message)) {
```

Comment:
Here I tried to be compatible with error messages I am not aware of. Do we have any spec of the possible error messages that can appear on the non-pods version?